components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2189,8 +2190,20 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
                 throw new Error(`No Stripe customer profile for '${attributionId}'`);
             }
 
-            await this.stripeService.setDefaultPaymentMethodForCustomer(customerId, setupIntentId);
-            await this.stripeService.createSubscriptionForCustomer(customerId, attributionId);
+            const createStripeSubscriptionOnUsage = await getExperimentsClientForBackend().getValueAsync(
+                "createStripeSubscriptionOnUsage",
+                false,
+                {
+                    user: user,
```

Comment:
We should supply the `team_id` as well for the case when it's enabled by a team

Reply:
Absolutely - will add it.